pkg/node/volumemgr.go

```diff
@@ -805,7 +805,9 @@ func (m *VolumeManager) discoverDataOnDrives() error {
 			}
 			continue
 		}
-		ll.Info(discoverResult.Message)
+
+		ll.Debug(discoverResult.Message)
```

Comment:
Do we really need to have this message? Isn't one on the line 802 enough?

Reply:
I have never seen log from the line 802